dockerfile_functions.sh

```diff
@@ -361,6 +361,21 @@ print_java_install_pre() {
        s390x) \\
          ESUM='$(sarray="${shasums}[s390x]"; eval esum=\${$sarray}; echo "${esum}")'; \\
          BINARY_URL='$(get_v3_binary_url "${JAVA_URL}")'; \\
+		EOI
```

Comment:
Whitespace seems odd for this whole code block...  Rest looks OK.

Reply:
Yup that is intentional unfortunately. The `if` and `cat` are tabbed and follow the indentation of the code block, the actual Dockerfile commands are spaced and need to be inline with the rest of the Dockerfile commands.